cost-analyzer/values.yaml

```diff
@@ -205,6 +205,50 @@ kubecost:
     #  cpu: "100m"
     #  memory: "256Mi"
 
+# Kubecost Metrics deploys a separate pod which will emit kubernetes specific metrics required 
+# by the cost-model. This pod is designed to remain active and decoupled from the cost-model itself. 
+# However, disabling this service/pod deployment will flag the cost-model to emit the metrics instead. 
+kubecostMetrics: 
+  # emitPodAnnotations: false 
+  # emitNamespaceAnnotations: false 
+  # emitKsmV1Metrics: true
+  
+  # Optional
+  # The metrics exporter is a separate deployment and service (for prometheus scrape auto-discovery)
+  # which emits metrics cost-model relies on. Enabling this deployment also removes the KSM dependency
+  # from the cost-model. If the deployment is not enabled, the metrics will continue to be emitted from
+  # the cost-model.
+  exporter:
+    enabled: false
+    image: "gcr.io/kubecost1/kube-metrics:v0.0.3-SNAPSHOT"
+    imagePullPolicy: Always 
+    port: 9005
+    resources: {}
+      # requests:
+      #  cpu: "200m"
+      #  memory: "55Mi"
+    ## Node tolerations for server scheduling to nodes with taints
+    ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+    tolerations: []
+
+    #  - key: "key"
+    #    operator: "Equal|Exists"
+    #    value: "value"
+    #    effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+    affinity: {}
+
+    # Service Monitor for Kubecost Metrics 
+    serviceMonitor:
+      enabled: false
+      additionalLabels: {}
```

Comment:
Is this disabled by default because service monitors are a Custom Resource?

Reply:
Yeah, a service monitor is for prometheus operator. We add support for it, but since it requires the existence of 3rd party tooling we don't include, it's disabled.